unit/src/MatrixToolsTest.C

```diff
@@ -41,15 +38,13 @@ MatrixToolsTest::matrixInversionTest1()
   mat2[2] = m[1][0] = 0.5;
   mat2[3] = m[1][1] = 1.0;
 
-  int error = MatrixTools::inverse(mat2, 2);
-  CPPUNIT_ASSERT(error == 0);
+  MatrixTools::inverse(mat2, 2);
   CPPUNIT_ASSERT_DOUBLES_EQUAL(1, mat2[0], 1E-5);
   CPPUNIT_ASSERT_DOUBLES_EQUAL(-2, mat2[1], 1E-5);
   CPPUNIT_ASSERT_DOUBLES_EQUAL(-0.5, mat2[2], 1E-5);
   CPPUNIT_ASSERT_DOUBLES_EQUAL(2, mat2[3], 1E-5);
 
-  error = MatrixTools::inverse(m, m);
-  CPPUNIT_ASSERT(error == 0);
+  MatrixTools::inverse(m, m);
```

Comment:
This is fine as long as you realize that if that `inverse()` throws it's going to cause the current timestep to fail and cutback `dt` and retry.

Is that the right thing to happen?